src/lib/services/workflow-service.ts

```diff
@@ -48,18 +48,13 @@ export const fetchAllWorkflows = async (
       `Error fetching workflows: ${err.status}: ${err.statusText}`;
   };
 
-  const handleError = () => {
```

Comment:
Is this case handled by ui-server now?

Reply:
when i tried setting a bad namespace and it would get processed by onError and not handleError

Reply:
also would it make sense to pass just a single error handler and process various cases in it instead of passing two error handlers. I think this would help understand the code easier vs trying to figure out the difference between `onError` and `handleError`

Reply:
![image](https://user-images.githubusercontent.com/11838981/192625470-4cc4b9cb-be5f-4288-bb57-af0ada701bcb.png)

Reply:
discussed directly, this change should be good

Then we also want to refactor some of error handling to completely remove the second error handler param